index.html

```diff
@@ -184,6 +187,19 @@ <h2> About <br> Myself </h2>
         </ul>
       </div>
     </section>
+
+    <footer class="contact rounded-corner shapes-bg-2">
+      <h2 id="contact">Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+      </p>
+      <form id="contact-form" action="https://formspree.io/f/xgedkalg" method="POST">
+        <input type="text" name="name" id="name" placeholder="Enter you name" aria-label="User name" value="Yeremias">
+        <input type="email" name="email" id="email" placeholder="your.email@domain.com" value="natajayanj14@gmail.com">
+        <textarea name="message" id="message" maxlength="500" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Please make sure all inputs(including textarea) have the required attribute, this will help activate the form validation on submit.
- [ ] Please ensure that the name input has a `maxlength="30"` attribute as stated in the requirements.
- [ ] Good job on including a placeholder. Kindly remove the value attribute from the inputs as the placeholder should handle it

Reply:
Great feedback, Anselem - thank you!